### DISCUSSION TITLE
background subtraction

### DISCUSSION THREAD

**Discussion by franke-biosaxs** [biosaxs-com/atsas-community#694] [Primary data processing]
On 2011-09-30 01:15:16, user Thermodynamics wrote:

SAXS background subtraction can be
I sample = I solution – I buffer (1)
Or
Isample= I solution –α* I buffer (2)
Then I guess equation (1)  is only used for very dilute solutions. and α in equation (2) should be dependent on solute concentration. Then how to determine α for any solute at certain concentration? I know for some proteins and nuclear acids, α has some correlations.
In addition, if the background is a binary solvent mixture and we only have their independent scattering, then how could the buffer scattering be? I guess it could be written as:
I buffer = a1*(I S1 –I tube ) + a2 *(I S2 –I tube )   (3)  (here empty tube scattering is subtracted )
and
I sample =(I solution –I tube )– α* I buffer (4)
a1 and a2 should be the volume fraction of respective solvent, since I is the differential scattering cross section per unit volume. If this is true, this means the scattering from a binary liquid mixture should necessarily be less than one the pure solvent scattering and bigger than the other. I am afraid this may not always be true.
In other words, if we have the scattering of the mixture and that of one of the solvents, we can estimate the scattering of another pure solvent as:
I s2 =((I buffer –I tube ) –a1*(I S1 –I tube ))/ a2    (5)

I hope I am clear on my concerns about background subtraction, since it is very important for further analysis. Besides, if the system is a two-phase system (solid-liquid suspension), will the equations change?

Thank you in advance for your kind help.

**Comment by franke-biosaxs** [comment]
On 2011-11-02 18:53:45, user konarev wrote:

Hi,

you can look up the standard equation for net sample scattering  in the paper from Strunz et.al. JAC (2000) V. 33, p.829
It mostly depends on the transmission signals of sample,  buffer and empty cell and is normally normalized by the concentration.

Cheers,
Peter

**Comment by franke-biosaxs** [comment]
On 2012-03-26 13:55:01, user rjain wrote:

Dear Peter,

I am interested to read publication from Strunz et al 2000. But I could not find the paper from the information provided by you.
Can you kindly provide more information for the same ?
Thanks in advance.
Regards,

Rohit

**Comment by franke-biosaxs** [comment]
On 2012-03-26 15:37:32, user franke wrote:


rjain wrote
> I am interested to read publication from Strunz et al 2000. But I could not find the paper from the information provided by you.

[Funnily, google can find it ...](http://lmgtfy.com/?q=strunz+journal+applied+crystallography+2000+v+33+829) - the third link in the list even points to a reprint ...

**Comment by franke-biosaxs** [comment]
On 2012-03-26 15:56:20, user rjain wrote:

I am sorry but I took naively JAC to be JACS paper.
And I was not able to get this paper without putting full name of journal.
But thanks a lot for taking out time and helping me.
It was really helpful.
Regards,
Rohit
